### PR TITLE
Added new features

### PR DESCRIPTION
**Pull request by Stanyke**

### CODE REVIEW

**Review by l-ezeh — APPROVED** [review]
This seem cute!

File: src/tranzakt.ts [hunk 1]
```diff
@@ -1,25 +1,42 @@
...
-    const response = await requestProcessor<Invoice>({
-      data: dynamicInvoice,
-      url: INVOICE_URL,
-      method: "POST",
-      headers: { "x-api-key": this.secretKey },
-    });
-    return response;
+  async getInvoiceDetails(invoiceId: string): Promise<Invoice> {
+    return this.invoiceService.getInvoiceDetails(invoiceId);
+  }
```
Comment (Dverybest) [inline src/tranzakt.ts:22]:
```suggestion
  async getInvoice(invoiceId: string): Promise<Invoice> {
    return this.invoiceService.getInvoiceDetails(invoiceId);
  }
```

File: src/tranzakt.ts [hunk 2]
```diff
@@ -1,25 +1,42 @@
...
+
+  async createInvoice(dynamicInvoice: CreateInvoiceProps): Promise<Invoice> {
+    return this.invoiceService.createInvoice(dynamicInvoice);
+  }
+
+  async invalidateAnInvoice(invoiceId: string): Promise<null> {
+    return this.invoiceService.invalidateAnInvoice(invoiceId);
+  }
```
Comment (Dverybest) [inline src/tranzakt.ts:30]:
```suggestion
  async invalidateInvoice(invoiceId: string): Promise<null> {
    return this.invoiceService.invalidateAnInvoice(invoiceId);
  }
```

File: src/tranzakt.ts [hunk 3]
```diff
@@ -1,25 +1,42 @@
...
+
+  async getCollectionDetails(collectionId: string): Promise<Collection> {
+    return this.collectionService.getCollectionDetails(collectionId);
+  }
```
Comment (Dverybest) [inline src/tranzakt.ts:34]:
```suggestion
  async getCollection(collectionId: string): Promise<Collection> {
    return this.collectionService.getCollectionDetails(collectionId);
  }
```

**Review by Dverybest — CHANGES_REQUESTED** [review]
Your PR name and branch name does not follow our standard please close and redo

**Pull request closed without merging**